Third review: Expanding README and proper pull request comment

tomasesquivelgc opened the pull request:
## Hello-Microverse project milestone: approved technical part

Every technical aspect of this project has been finished. Last commits include:
  - Linters passing every test
  - README.md is up to par with Microverse standards
  - test.md is successfully ignored (.gitignore file is working as intended)
As well as non technical issues regarding GitHub Flow have been fixed

### In this new review
The following has been implemented:
  - PR summary

tomasesquivelgc committed:
removed test.md from workflows folder

tomasesquivelgc committed:
created test.md  on the main repo

LionRouge1 reviewed (changes requested):
Hi @tomasesquivelgc, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![almost](https://media.giphy.com/media/R2m2NzVxQ3pbG/giphy.gif)

## Highlights
- Your commit messages are meaningful :1st_place_medal: 
- Good use of Github flow :+1: 
## Required Changes ♻️
- [x] Please, make sure linter is set up correctly. Because having linters correctly set up in your project will help ensure that the quality of your written code is good.
- [x] Please add a summary describing in more detail the changes introduced in the project. This will help other developers know what to expect from your project and it also makes your project look more professional.
- I think your introduction session in your README.md file, will be a great fit for this PR summary. Or what do you think?
> This is my repository for the "Hello-microverse" project In this project I am putting to the test my ability to:
    - create and merge branches
    - add linters to my repo
    - add .gitignore files

- [x] Kindly, use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project. It is more professional and contains all the necessary information that visitors or recruiters are looking for when they visit your project.
- [x] Good job adding the 3 files as required in the project requirement :clap: However `.html`  will be considered as a hidden file by operating systems, so kindly add `.html` as a prefix to any name. eg: `index.html`, `home.html`, etc...
- [x] Please, make sure that  the `test.md` file is actually ignored by git and is not present in your GitHub repository as required in the project requirement.
![Screenshot from 2023-03-21 22-46-21](https://user-images.githubusercontent.com/91142432/226763109-d0144c12-b574-448f-a1b4-f8a84f4919d0.png)
- To accomplish that, kindly, add `test.md` to your `.gitignorre` file, then it will look like this:
```sh
  node_modules/

  test.md
```

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [OPTIONAL] **IMPORTANT NOTE:** Please, make sure to create a **Pull Request** only if you have completed implementing the project requirement and your project is ready for review. Follow [these steps](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/how_to_ask_for_a_code_review.md) to request a code review of your project.
- In this activity, you need to create **one** Pull Request containing all the changes you made. It will help Code reviewers to use GitHub features when reviewing your project so you can easily understand the context of each requirement and suggestion left by the reviewer.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tomasesquivelgc committed:
added name to .html file

the html file is now called "index.html" for easier visualization

tomasesquivelgc committed:
added test.md to .gitignore

now test.md should be ignored

tomasesquivelgc committed:
linters succesfully installed

style.css has been corrected using linters

tomasesquivelgc committed:
expanded README.md and corrected package.json

added recent changes to README introduction.
fixed upper-cases into lower-cases in package.json

aamirkhan2478 commented on README.md:
- [x] As mentioned by the previous reviewer :point_up: use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project. It is more professional and contains all the necessary information that visitors or recruiters are looking for when they visit your project but your `README` file is not the same as Microverse provided `README` template so kindly use the same as Microverse provided `README` template and also keep it mind in this template only `FAQ` is optional other sections are required. :+1:

aamirkhan2478 reviewed (changes requested):
Hi @tomasesquivelgc , 

![](https://media3.giphy.com/media/RcZlXVml8oKtoGQeqe/giphy.gif?cid=ecf05e47cw5pjd1gapdzk1t3vp0n73caujoqxw3vkcu8ssng&rid=giphy.gif&ct=g)

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

 ## Highlights
✔️ Good Effort
✔️ Your design is good.

 ## Required Changes ♻️
- [ ] As mentioned by the previous reviewer :point_up: Please add a PR summary and PR title describing in more detail the changes introduced in the project. This will help other developers know what to expect from your project and it also makes your project look more professional. Below is an example of a descriptive PR summary and PR title so kindly check it and edit them. :slightly_smiling_face: 
![219520522-f0a8c7ba-b272-4a26-81c7-c233913f9c1e](https://user-images.githubusercontent.com/45226848/227026971-e4b06bb0-4928-42ff-b6d4-79756f391621.png)

- [x] Kindly fix your conflict by watching this [video](https://www.youtube.com/watch?v=mOJazBNrG-c) and resolve this conflict. :+1: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were missed by the previous reviewer and addressing them will really improve your application._

[OPTIONAL] This is optional but recommended your `branch` name is not descriptive kindly edit your `branch` name should be any feature or action like `add-heading` so kindly don't skip this optional suggestion and also watch this [video](https://www.youtube.com/watch?v=LYJ81OKXH9c&t=232s) for how to change your `branch` name :slightly_smiling_face: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tomasesquivelgc committed:
Merge branch 'main' into creating-test.md-

tomasesquivelgc committed:
Added LICENSE and updated README

README file has been changed to fit a professional standard.
LICENSE file has been added and linked in the README

tomasesquivelgc commented:
## Third review
### Changes to this branch are:
- Renamed the branch to fit the stage I'm in
- README redone to fit the template
- Comments added for better communication and Flow

tomasesquivelgc committed:
Linters version changed from 18x to 12x

tomasesquivelgc committed:
webhint node version changed from 12 back to 18

this is to find a solution to the webhint error.

tomasesquivelgc committed:
added "recent changes" to README introduction

iambenkis reviewed (changes requested):
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @tomasesquivelgc 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [ ] Good job setting up your first project at microverse 👍 , but   As mentioned by the previous reviewer ☝️ Please add a PR summary and PR title describing in more detail the changes introduced in the project. This will help other developers know what to expect from your project and it also makes your project look more professional. Below is an example of a descriptive PR summary and PR title so kindly check it and edit them. In your case, there is not PR description 
![pr](https://user-images.githubusercontent.com/80458199/227238065-e8b32e4c-b265-4f5e-9359-39b22baa1493.JPG)
 you may us this exemple
![pr](https://user-images.githubusercontent.com/80458199/227238376-7c078637-89f8-451a-9111-57c3af042475.png) 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

LionRouge1 reviewed (approved):
Hi @tomasesquivelgc , 

# STATUS: APPROVED! 🥇 💯 
![done](https://media.giphy.com/media/l1uga8ObPBnIBoXUQ/giphy.gif)

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
